setup.py

```diff
@@ -41,6 +42,10 @@ def _debugInstance(x):
         print("{}:     {}".format(a, getattr(x, a)))
 
 
+def joinArgs(args):
```

Comment:
We're using snake case in python for functions:
```suggestion
def join_args(args):
```